src/linker/pipeline.py

```diff
@@ -78,12 +78,12 @@ def get_input_files(self, implementation: Implementation) -> list:
     def get_output_dir(self, implementation: Implementation) -> Path:
         idx = self.implementation_indices[implementation.name]
         if idx == len(self.implementations) - 1:
-            return self.config.results_dir
+            return Path()
```

Comment:
Where does this write the outputs? The working directory?

Reply:
The working directory *of snakemake*, which is what we call the results directory. That is, we told snakemake to make its working directory self.config.results_dir, and when we write to the snakefile, all paths must be relative to that.